3_visualize.R

```diff
@@ -2,147 +2,217 @@ source('3_visualize/src/drought_learner_viz_components.R')
 
 p3_targets <- list(
        
+  
+  # Global design variables to be used in each frame, with "dv_" as prefix for "design variable"
+  tar_target(p3_dv_tibble,
```

Comment:
I think this new design variable tibble is really nice to have, and makes the design easy to tweak, which is great.

SC: One downside to this approach is that any time anything in the tibble changes, everything that uses it downstream will get rebuilt. I don't think that's particularly a problem here, since the targets build so quickly, but I think it's an argument for considering either a) splitting all of these into individual targets (tedious and would mean tons of arguments to the plotting functions, as you noted), or b) splitting this into a few separate tibbles, e.g.,  splitting this into 5 tibble to match the headings you've included. That way, all of the frames wouldn't be rebuilt if you wanted to change the outline on the zoom box in frame e.

Reply:
Added new issue #32